guides/20240819_ai_transcription.md

```diff
@@ -0,0 +1,312 @@
+---
+title: "Transcribe Video and Audio with Python"
+description: "This guide provides a comprehensive step-by-step process for setting up and using Sapat to automatically transcribe video and audio files using Azure OpenAI's Whisper API."
+date: 2024-08-19
+author: "Jacob Gaffke"
+---
+
+# Transcribe Video and Audio with Python
+
+## Introduction
+
+Transcribing video and audio files is essential in various fields, from content creation to legal documentation and academic research. Manual [transcription](/definitions/transcription.md) is labor-intensive and error-prone, making automation an invaluable tool. This guide introduces [a Python-based tool](https://github.com/nkkko/sapat) that automates the [transcription](/definitions/transcription.md) of video files by converting them into MP3 format, transcribing the audio using one of a variety of supported advanced AI models, and saving the [transcription](/definitions/transcription.md) as a text file. Whether you're a content creator, a researcher, or someone needing to convert speech into text, this guide will help you accomplish your tasks efficiently.
+
+By following this guide, you will learn how to set up the necessary environment, install the required software, and use the [transcription](/definitions/transcription.md) tool. The tool supports processing individual video files or entire directories, ensuring temporary MP3 files are cleaned up after [transcription](/definitions/transcription.md). You can choose between using Azure OpenAI's Whisper API or Rev AI API based on your preference and access. A basic understanding of Python and command-line operations, as well as access to the relevant API, is required.
+
+### Prerequisites
+
```

Comment:
only prereq should be daytona and repo with appropriate devcontainer that will set all of the prereq which I have in the repo. 

with every article the main idea is to show how daytona simplifies development, quick hacking/prototyping, oss contributions, etc.